docs/reference/operator_parameters.md

```diff
@@ -343,8 +343,13 @@ configuration they are grouped under the `kubernetes` key.
   considered `ready`. The operator uses values of those labels to detect the
   start of the Kubernetes cluster upgrade procedure and move master pods off
   the nodes to be decommissioned. When the set is not empty, the operator also
-  assigns the `Affinity` clause to the Postgres pods to be scheduled only on
-  `ready` nodes. The default is empty.
+  assigns the `nodeAffinity` clause to the Postgres pods to be scheduled only
+  on `ready` nodes. If a `nodeAffinity` is specified in the postgres cluster
+  manifest as well the `nodeSelectorTerms` will get merged. If the 
+  `nodeAffinity` of the manifest contains only one `matchExpressions` slice
+  the node readiniess label expressions will be moved there (AND condition).
```

Comment:
1. typo in `readiniess`
2. `be moved there` reads a bit misleading. I assume the original manifest is not modified, right ? But internally in the operator the `matchExpressions` slice will contain the node readiness label